extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -748,8 +796,8 @@ export class AmpAnalytics extends AMP.BaseElement {
    * Merges two objects. If the value is array or plain object, the values are
    * merged otherwise the value is overwritten.
    *
-   * @param {Object|Array} from Object or array to merge from
-   * @param {Object|Array} to Object or Array to merge into
+   * @param {(Object|Array)} from Object or array to merge from
```

Comment:
Removed.
(The "why" is that at Taymon's suggestion, I tried this when I was experiencing all those type-checking errors last week in files that I didn't touch. They didn't fix it, and I later found something else which did, but did not remove these parens.)